frontend/packages/console-app/src/components/cloud-shell/CloudShellTab.scss

```
@@ -0,0 +1,16 @@
+.co-cloud-shell-tab {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  background-color: var(--pf-global--Color--dark-100);
+  &__header {
```

This styling should be applied to the `CloudShellTabHeader` instead of here.
```suggestion
  &__header {
    flex-shrink: 0;
```

---

updated..Removed CloudShellheader